test/downstream/delaydiffeq.jl

```diff
@@ -31,3 +31,19 @@ DDEProblemLibrary.importddeproblems()
         @test sol[1, :] ≈ sol_scalar.u
     end
 end
+
+function lotka_volterra!(du,u,h,p,t)
+    🐰, 🐺 = u
+    α,β,γ,δ,τ = p
+    🕥🐰 = h(p,t-τ;idxs=1)
+    du[1] = d🐰 = α*🕥🐰 - β*🐺*🐰
+    du[2] = d🐺 = γ*🐺*🐰 -δ*🐺
+    nothing
+end
+uₒ = [1.0,1.0]
+tspan = (0.0, 10.0)
+h(p,t) = [1.0,1.0]
+h(p,t;idxs = 1) = 1.0
+p = [1.5,1.0,3.0,1.0,1.0]
+prob = DDEProblem(lotka_volterra!,uₒ,h,tspan,p,constant_lag = (p[end],))
```

Comment:
Sorry, noticed too late that there was also a typo here:
```suggestion
prob = DDEProblem(lotka_volterra!,uₒ,h,tspan,p,constant_lags = (p[end],))
```